src/css/typography.css

```diff
@@ -1,9 +1,9 @@
 @import "fonts.css";
 
 :root {
-	--font-size-50: 0.685 rem;
-	--font-size-75: 0.75 rem;
-	--font-size-100: 0.875 rem;
+	--font-size-50: 0.685rem;
+	--font-size-75: 0.75rem;
+	--font-size-100: 0.875rem;
```

Comment:
These styles weren't previously applied properly because of the space between the value and rem